app/src/main/java/com/lagradost/cloudstream3/services/PackageInstallerService.kt

```diff
@@ -29,7 +29,7 @@ class PackageInstallerService : Service() {
 
     private val baseNotification by lazy {
         val flag = if (SDK_INT >= Build.VERSION_CODES.S) {
-            PendingIntent.FLAG_MUTABLE
+            PendingIntent.FLAG_IMMUTABLE
```

Comment:
Just noting I'm not positive if this one here needs changed also or not.

Reply:
This should work though from examining things. Intent is never changed si IMMUTABLE should be the correct option here, theoretically...